### DISCUSSION TITLE
Identical results problems

### DISCUSSION THREAD

**Discussion by Ali141272** [neperfepx/neper#797] [Q&A]
Hello Professor Dr. @rquey,
I hope you will be fine. Professor, I tried a lot to make the exact microstructure but couldn't get the same results and there is a little bit difference. Professor, please have a look at the attached pictures, in which I mentioned both models (our model and literature model). You will find quite difference in these two models.
My models commands as follows:
For Fig-A, I have used this command;
neper -T -n 100 -morpho gg -reg 1
neper -T -n 100 -morpho "diameq:lognormal(1,0.032),1-sphericity:lognormal(0.0123,0.03)" -domain "square(0.05,0.1)" -dim 2
For Fig-B, I have used a command;
neper -T -n 100 -morpho gg -reg 1
neper -T -n 100 -morpho "diameq:lognormal(1,0.06),1-sphericity:lognormal(0.0123,0.03)" -domain "square(0.05,0.1)" -dim 2

Professor, please give me suggestions what I need to do through which I get the same model.

I am waiting for your kind response.

Thank you.

![A](https://github.com/neperfepx/neper/assets/102291665/b3a9aa8a-fa3f-445c-b3e5-1884c816c75b)

**Comment by rquey** [comment]
Looks like the figures on the right involve smaller grain sphericities, so:
```
neper -T -n 100 -morpho "diameq:lognormal(1,0.032),1-sphericity:lognormal(0.123,0.03)" -domain "square(0.05,0.1)" -dim 2
neper -V n100-id1.tess -dataedgerad 0.0001 -print d
```
![d](https://github.com/user-attachments/assets/0ef2b42c-9454-4c46-b372-3417c2ac3f9f)

You'd need to adjust the actual values.
